src/main.js

```diff
@@ -16,6 +16,8 @@ var makeYourOwnBtn = document.querySelector('.show-form');
 var takeMeBackBtn = document.querySelector('.show-main');
 var showSavedPostersBtn = document.querySelector('.show-saved');
 var backToMainBtn = document.querySelector('.back-to-main');
+var showMyPosterBtn = document.querySelector('.make-poster');
+showMyPosterBtn.setAttribute('type', 'button');
```

Comment:
I'm glad we realized the need to assign the type="button" as this button is nested inside of a form, which changes its default behavior. Yey!